.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish Python 🐍 distributions 📦 to PyPI
+
+on:
+  release:
+    types: [published]
+jobs:
+  build-and-publish-to-pypi:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python 3.6
```

Comment:
Similarly here, do you want to use `actions/checkout@v2` instead?

Reply:
No idea, should I? :-)

Reply:
It's just a matter of compatibility when it comes to use a fixed major version instead of main branch. I would suggest to use a fixed major version if possible.